pyphare/pyphare/pharesee/plotting.py

```diff
@@ -103,7 +103,7 @@ def dist_plot(particles, **kwargs):
         ax.set_xlim(kwargs["xlim"])
     if "ylim" in kwargs:
         ax.set_ylim(kwargs["ylim"])
-    ax.legend()
+    #ax.legend()
```

Comment:
no legend ?

Reply:
well, as is, i do not see any labels to write in the legend... and this legend gives me a warning. as i do not see the interest of this, i commented it.

Reply:
because i'm not comfortable with removing something I did not wrote, since I'm not totally sure it has to...

Reply:
that's what the review is for. Go for it.
I think this ax.legend() is what drew the little gray square on the top right part of the figure in : https://github.com/PHAREHUB/PHARE/pull/355